databox/tsconfig.json

```diff
@@ -10,6 +10,6 @@
     "allowJs": true,
     "noImplicitOverride": true
   },
-  "include": ["./**/*.ts", "./**/*.json", ".eslintrc.js"],
+  "include": ["./**/*.ts", "./**/*.json", ".eslintrc.js", "examples/basic.mjs"],
```

Comment:
Won't typescript try to convert this file if you include like this?

Reply:
The default "Copyfiles" that runs post build should automatically copy .mjs files, i think.

Reply:
Shoot. I think my VSCode auto renamed basic.js to basic.mjs. That's not right. I'll push a small fix to main.